helm-chart/renku/values.yaml

```diff
@@ -73,6 +73,26 @@ global:
     jena:
       ## specify which dataset should be used in Jena
       dataset:
+  externalServices:
+    # If an external Postgres database is defined in global.externalServices.postgresql,
+    # global.externalServices.postgresql.enabled should be true, and postgresql.enabled should be false.
+    # By default, Renku-bundled Postgres is enabled.
+    postgresql:
+      enabled: false
+      # host:
+
+      ## We use the defaults here.
+      maintenanceDatabase: postgres
+      username: postgres
+
+      ## The admin password should be set explicitly, otherwise a random string will be created.
+      ## Alternatively an existing secret can be provided. Note that postgres
```

Comment:
I disagree that we should have the same names in both sections. External Postgres has nothing to do with Bitnami, so I don't know why we should match external Postgres variable names to their decisions on names.

`maintenanceDatabase` is taken from pgAdmin, which I think is a good name for it.
![SCR-20230214-ee7](https://user-images.githubusercontent.com/515042/218693161-d3e2265f-7055-4081-8e2d-80b5477a11d3.png)

Reply:
I have never used `pdAdmin` and I was really confused what `maintenanceDatabase` means. I know the external postgres has nothing to do with Bitnami. But we are presenting in our values file two sets of configurations. The one for bitnami and the one for an external db. They are both for postgres and they both refer to the same part of the configuration or postgres. So it is really weird that the settings that essentially mean and point to the same thing (i.e. the postgres/maintenance database) have 2 completely different names in the values file. If someone is looking at this for the first time and deciding whether to use the built in or external postgres it is beneficial to be able to map and understand between the two sections. It is the same case for moving from a built in to external postgres. If someone did this they would have to figure out that the value at `postgresql.postgresqlDatabase` is analogous to the value `global.externalServices.MaintenanceDatabase`. So the values are not completely unrelated and understanding of how they are related is important in several cases. That is why we should make the connection easier to see not harder.

Reply:
I think the name`maintenanceDatabase` makes it clear that this is not the database that Renku will actually use to store data in, as opposed to `postgresDatabase`, which to me is less clear. I also think we'd be making it harder for ourselves in the future by having to conform to whatever Bitnami decides to do with their Helm chart values. We're just making more work for ourselves in the future this way.

Reply:
okay, I did some testing:  #2953 and we simply don't need this value. It can also be removed from the init scripts.
What we need is `postgres.postgresqlDatabase` (which this PR already is preserving), only to enable the metrics when deploying Postgres as "internal" service.

Remove this value and from the init scripts.
End of the discussion.